tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -967,12 +967,13 @@ public static void test_duckdb_getObject_with_class() throws Exception {
         stmt.execute("CREATE TABLE b (vchar VARCHAR, bo BOOLEAN, sint SMALLINT, nint INTEGER, bigi BIGINT,"
                      + " flt FLOAT, dbl DOUBLE, dte DATE, tme TIME, ts TIMESTAMP, dec16 DECIMAL(3,1),"
                      + " dec32 DECIMAL(9,8), dec64 DECIMAL(16,1), dec128 DECIMAL(30,10), tint TINYINT, utint UTINYINT,"
-                     + " usint USMALLINT, uint UINTEGER, ubig UBIGINT, hin HUGEINT, blo BLOB)");
+                     + " usint USMALLINT, uint UINTEGER, ubig UBIGINT, hin HUGEINT, uhin UHUGEINT, blo BLOB)");
         stmt.execute(
             "INSERT INTO b VALUES ('varchary', true, 6, 42, 666, 42.666, 666.42,"
             +
             " '1970-01-02', '01:00:34', '1970-01-03 03:42:23', 42.2, 1.23456789, 987654321012345.6, 111112222233333.44444, "
-            + " -4, 200, 50001, 4000111222, 18446744073709551615, 18446744073709551616, 'yeah'::BLOB)");
+            +
+            " -4, 200, 50001, 4000111222, 18446744073709551615, 18446744073709551616, 18446744073709551616, 'yeah'::BLOB)");
```

Comment:
I think `18446744073709551616` is used here for HUGEINT because it surpasses the max of UBIGINT, I believe `170141183460469231731687303715884105728` would surpass the max for HUGEINT so this would be a better test here for UHUGEINT